README.md

```diff
@@ -15,6 +15,8 @@ Python podman library.
 
 Provide a stable API to call into.
 
+**Notice:** When Podman version 2.0 is released in 2020, it will include a new REST API that will replace the varlink interface.  At the time of the release of Podman v2.0, the varlink interface will be deprecated, and at a later release removed from Podman.  Starting with Podman v1.9.0, the REST API is available for test usage.  The documentation for the REST API resides [here](http://docs.podman.io/en/latest/_static/api.html#operation/changesContainer).
```

Comment:
It's actually the v1.8.0 release where `podman system service` became available, albeit in alpha form.